data/chain-store/sge-network-testnet.json

```
@@ -0,0 +1,57 @@
+{
+  "rpc": "https://rpc.staging.sixsigmasports.io",
+  "rest": "https://api.staging.sixsigmasports.io",
+  "chainId": "stage-sgenetwork",
+  "chainName": "stage-sgenetwork",
+  "image": "https://app-assets.testnet.sixsigmasports.io/img/logo-center.png",
```

We have uploaded the SGE logo to our CDN, here's the change you can make -

```suggestion
  "image": "https://assets.leapwallet.io/sge.png",
```